packages/fx-core/src/common/samples-config.json

```diff
@@ -162,6 +162,16 @@
             "time": "5min to run",
             "configuration": "Ready for local debug",
             "suggested": false
+        },
+        {
+            "id": "graph-connector-app",
+            "title": "Graph Connector App",
+            "shortDescription": "Showcases how to build custom Graph Connector",
+            "fullDescription": "This sample app showcases how to build custom Graph Connector with Azure Functions and query data using Microsoft Graph Client and TeamsFx SDK.",
+            "tags": ["Tab", "Azure function", "TS", "SSO"],
+            "time": "10min to run",
+            "configuration": "Ready for local debug",
```

Comment:
Is it ready for local debug? I think it needs manual steps to consent the permission first?

Reply:
Updated:
![image](https://user-images.githubusercontent.com/1050213/164956585-c50a6f40-bdaf-4df8-9b6f-c49ab2658d60.png)